everyvoice/tests/test_configs.py

```diff
@@ -1,13 +1,20 @@
 import json
-import tempfile
 import time
 from pathlib import Path
+from tempfile import TemporaryDirectory
 from typing import Callable
 from unittest import TestCase
 
 import yaml
 from pydantic import ValidationError
 
+try:
+    from readalongs.portable_tempfile import (
+        PortableNamedTemporaryFile as NamedTemporaryFile,
+    )
+except ImportError:
+    from tempfile import NamedTemporaryFile
+
```

Comment:
Is there a way not to require readalongs?  Is EV already dependent on readalongs?  Wouldn't it be wiser to have that `readalongs.protable_temfile` be a new library that both EV and readalongs could import?

Reply:
We don't depend on readalongs yet, and with the try/except we still don't really, which is why I did not add it to requirements.

Also, when we do #439, we will then really add the readalongs dependency to support that export feature, which is in part why I thought I could slide this weird thing in.